mesa/model.py

```diff
@@ -57,6 +57,9 @@ def __new__(cls, *args: Any, **kwargs: Any) -> Any:
             # advance.
             obj._seed = random.random()  # noqa: S311
         obj.random = random.Random(obj._seed)
+        # TODO: Remove these 2 lines after Mesa 2.3
```

Comment:
Should be before Mesa 3.0, we can't know if 3.0 comes after 2.3

Reply:
Fixed.